dashboard/app/models/pd/workshop.rb

```diff
@@ -292,6 +301,10 @@ def course_name
     COURSE_NAMES_MAP[course]
   end
 
+  def course_target
+    COURSE_URLS_MAP[course] if COURSE_URLS_MAP.key? course
```

Comment:
nit: you don't need the `if` clause. `[key]` for a nonexistent key returns that Hash's default value, which by default (as when it's defined with `{...}`) is nil.

Or:
```
irb(main):001:0> {x: 1, y: 2}[:z]
=> nil
```